### DISCUSSION TITLE
unexpected failures in the artillery Run

### DISCUSSION THREAD

**Discussion by ChetanKB** [artilleryio/artillery#1454] [Artillery]
Hi Everyone,
I am getting an error like this errors.ERR_GOT_REQUEST_ERROR for the artillery run, But API's working as expected I didn't see any issues in the API I have checked  Manually and verified, and also I have tested with the Rest-assured there I am not getting any error. 
While executing in the artillery run only I am facing the issue.

**Comment by hassy** [comment]
hi @ChetanKB 👋 can you provide more information about what your scenario is doing?

**Reply by ChetanKB** [reply]
Hi @hassy 
we have only one scenario in which we are doing multiple curd operations. previously it was working fine but now we are facing the issues and also there is no changes have been done in the code.
 These are the local metrics we are getting
![MicrosoftTeams-image](https://user-images.githubusercontent.com/53107584/170306509-f338b260-6ef8-47aa-bac0-75679c1db3d5.png)
